bin/create-php-parser.js

```diff
@@ -5,7 +5,7 @@ const phpegjs = require( 'phpegjs' );
 const fs = require( 'fs' );
 const path = require( 'path' );
 
-const peg = fs.readFileSync( 'blocks/api/post.pegjs', 'utf8' );
+const peg = fs.readFileSync( 'packages/grammar/gutenberg.pegjs', 'utf8' );
```

Comment:
The thing with the current grammar is that it's simultaneously written to be able to generate JS and PHP parsers, which makes it a somewhat specific or purposeful grammar (note that we can generate a [generic](https://user-images.githubusercontent.com/150562/38624667-98ccbc98-3da0-11e8-8a60-026990052879.png) — that is, with no code — grammar from it as of #6116).

There's nothing wrong with importing from the package to create the PHP parser, though. I'm mostly just thinking aloud. But it is a fact that core Gutenberg now depends on a package which sounds relatively abstract ("grammar") but is actually written with PHP-specific rules.

Reply:
I think it's fine personally, we just have to publish the `pegjs` as well to make it part of the "official" API of the package.